steveify1/nrc-hr-backend
# chore(migrations):create and seed tables

#### What does this PR do?
create and seed tables
#### Description of Task to be completed?
######  Create tables for the following tables models
* Skills
* Qualifications
* Pensionables
* Countries
* EmployeeSkills
* EmployeeQualifications
* EmployeeStatuses

###### Seed the following tables
* Skills
* Qualifications
* Pensionables
* Countries
* EmployeeStatuses
#### How should this be manually tested?
* clone the repository
* cd into the project directory
* navigate to `src/models` to see the newly generated models
* navigate to `src/migrations` to see the newly generated migration files
* navigate to `src/seeders` to see the newly generated seeder files
#### Any background context you want to provide?
#### What are the relevant pivotal tracker stories?
#84, #85, #86, #125, #126, #129, #130, #131 
#### Screenshots (if appropriate)
#### Questions:
